apps/gp2-server/src/data-providers/project.data-provider.ts

```diff
@@ -104,13 +104,21 @@ export function parseProjectToDataObject(
       [],
     ) || [];
 
+  if (project.keywords && !project.keywords.every(gp2.isProjectKeyword)) {
+    throw new TypeError('Invalid keyword received from Squidex');
+  }
+
   return {
-    id: project.id,
-    title: project.flatData.title || '',
-    startDate: project.flatData.startDate || '',
-    endDate: project.flatData.endDate || undefined,
-    status: project.flatData.status,
-    projectProposalUrl: project.flatData.projectProposal || undefined,
+    id,
+    title: project.title || '',
+    startDate: project.startDate || '',
+    endDate: project.endDate || undefined,
+    status: project.status,
+    projectProposalUrl: project.projectProposal || undefined,
+    pmEmail: project.pmEmail || undefined,
+    leadEmail: project.leadEmail || undefined,
+    description: project.description || undefined,
```

Comment:
I've defaulted the description to `''` in working groups and have it as a mandatory string. not sure which is the right approach, we should check and make both the same

Reply:
on this, it's not mandatory